[2025-01-24T22:37:23.682+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T22:37:23.697+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskA manual__2025-01-24T22:37:21.881015+00:00 [queued]>
[2025-01-24T22:37:23.704+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_multiple_tasks.taskA manual__2025-01-24T22:37:21.881015+00:00 [queued]>
[2025-01-24T22:37:23.706+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-24T22:37:23.718+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): taskA> on 2025-01-24 22:37:21.881015+00:00
[2025-01-24T22:37:23.724+0000] {standard_task_runner.py:72} INFO - Started process 825 to run task
[2025-01-24T22:37:23.726+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'executing_multiple_tasks', 'taskA', 'manual__2025-01-24T22:37:21.881015+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/execute_multiple_tasks.py', '--cfg-path', '/tmp/tmpfeipgqdq']
[2025-01-24T22:37:23.728+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask taskA
[2025-01-24T22:37:23.764+0000] {task_command.py:467} INFO - Running <TaskInstance: executing_multiple_tasks.taskA manual__2025-01-24T22:37:21.881015+00:00 [running]> on host 6eaa7309f7d4
[2025-01-24T22:37:23.826+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='executing_multiple_tasks' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2025-01-24T22:37:21.881015+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-24T22:37:21.881015+00:00'
[2025-01-24T22:37:23.828+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T22:37:23.837+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-24T22:37:23.839+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK A has executed!']
[2025-01-24T22:37:23.849+0000] {subprocess.py:99} INFO - Output:
[2025-01-24T22:37:23.851+0000] {subprocess.py:106} INFO - TASK A has executed!
[2025-01-24T22:37:23.852+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-24T22:37:23.874+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T22:37:23.876+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=executing_multiple_tasks, task_id=taskA, run_id=manual__2025-01-24T22:37:21.881015+00:00, execution_date=20250124T223721, start_date=20250124T223723, end_date=20250124T223723
[2025-01-24T22:37:23.901+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T22:37:23.914+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-24T22:37:23.919+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
